src/main/java/mate/academy/DataGenerator.java

```diff
@@ -0,0 +1,271 @@
+package mate.academy;
+
+import java.time.LocalDateTime;
+import java.time.Month;
+import java.util.ArrayList;
+import java.util.List;
+import mate.academy.exception.AuthenticationException;
+import mate.academy.exception.RegistrationException;
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
+import mate.academy.model.Movie;
+import mate.academy.model.MovieSession;
+import mate.academy.model.User;
+import mate.academy.security.AuthenticationService;
+import mate.academy.service.CinemaHallService;
+import mate.academy.service.MovieService;
+import mate.academy.service.MovieSessionService;
+import mate.academy.service.OrderService;
+import mate.academy.service.ShoppingCartService;
+
+public class DataGenerator {
+    private static Injector injector = Injector.getInstance("mate.academy");
+    private static Movie casinoRoyale;
+    private static Movie bulletTrain;
+    private static Movie maverick;
+    private static CinemaHall imaxHall;
+    private static CinemaHall hallWith4D;
+    private static MovieSession bulletTrainSession;
+    private static MovieSession bulletTrainSession1;
+    private static MovieSession bulletTrainSession2;
+    private static MovieSession maverickSession;
+    private static MovieSession maverickSession1;
+    private static MovieSession maverickSession2;
+    private static MovieSession casinoRoyaleSession;
+    private static MovieSession casinoRoyaleSession1;
+    private static MovieSession casinoRoyaleSession2;
+    private static User maxine;
+    private static User danice;
+    private static MovieService movieService;
+    private static CinemaHallService cinemaHallService;
+    private static MovieSessionService movieSessionService;
+    private static AuthenticationService authenticationService;
+    private static ShoppingCartService shoppingCartService;
+    private static OrderService orderService;
+
+    static {
+        movieGenerator();
+        cinemaHallGenerator();
+        movieSessionGenerator();
+        movieServiceGenerator();
+        cinemaHallServiceGenerator();
+        getUsersEmails();
+        movieSessionServiceGenerator();
+        try {
+            registerUsers();
+            loginUsers();
+        } catch (RegistrationException | AuthenticationException e) {
+            throw new RuntimeException("Can't use register or login user", e);
+        }
+        shoppingCartGenerator();
+        orderGenerator();
+    }
+
+    public static List<String> getUsersEmails() {
+        maxine = new User();
+        maxine.setEmail("maxine@gmail.com");
+        maxine.setPassword("nothingPassword");
+        danice = new User();
+        danice.setEmail("danice@gmail.com");
+        danice.setPassword("qwertyghj");
+        List<String> emails = new ArrayList<>();
+        emails.add(maxine.getEmail());
+        emails.add(danice.getEmail());
+        return emails;
+    }
+
+    private static void movieGenerator() {
```

Comment:
can we return list of movies and use `movieGenerator` inside `movieServiceGenerator`?
so we could remove `casinoRoyale`, `bulletTrain` and other static fields in `DataGenerator`
do this for other methods as well